lib/canvas.js

```diff
@@ -0,0 +1,140 @@
+"use strict";
+/* global require, module, d3 */
+
+var configurable = require('./util/configurable');
+
+module.exports = function (d3) {
+
+	return function (config) {
```

Comment:
this file is not needed for now, I wanted to try to separate the code for the canvas from the one used for the svg, but I couldn't make it work separately